common:
  lidar_topic:  "/robot/dlio/odom_node/pointcloud/deskewed"                  # From FASTLIO2
  lidar_color_topic:  "/livox/color_lidar" #"/color_lidar"      # Pub color pointcloud
  odom_topic:  "/robot/dlio/odom_node/odom"                             # From FASTLIO2
  camera_topic:  "/camera/color/image_raw"
  frame_id: "camera_init"                                   # From FASTLIO2 's  frame_link

mapping:
  extrinsicT: [0.000785119,-0.999997,-0.00246143,-0.0376593
              0.0361507,0.0024882,-0.999343,-0.0597089
              0.999346,0.000695621,0.0361526,0.121811
              0,0,0,1]        #  lidar2camera 外参  From livox_camera_calib

  intrisicT: [ 1858.851674, 0.000000, 689.121011,
                0.000000, 1850.424325, 525.758177,
                0.000000, 0.000000, 1.000000]           # camera 内参

  ditortion: [-0.025024, -0.068101, -0.001616, -0.004414, 0.000000]      # camera 畸变系数

savepcd:
  savePCDDirectory: "/catkin_ws/src/livox_color/PCD/"
